spec/hydra/pcdm/models/object_spec.rb

```diff
@@ -14,6 +14,45 @@
     end
   end
 
+  context 'when aggregated by other objects' do
+
+    before(:all) do
+      @collection1 = Hydra::PCDM::Collection.create
```

Comment:
... and there's not actually need for isolation between the tests.  They're all testing idempotent behavior.